sale_coupon_multi_use/models/sale_coupon.py

```diff
@@ -64,32 +55,25 @@ def _get_related_sale_order_line(self, sale_order):
             lambda r: r.product_id == discount_product
         )[0]
 
-    def _prepare_consumption_line(self, amount_consumed, sale_order):
+    def _prepare_consumption_line(self, sale_order_line):
         self.ensure_one()
-        line = self._get_related_sale_order_line(sale_order)
+        # line = self._get_related_sale_order_line(sale_order)
```

Comment:
Maybe you could just forget this statement.

Reply:
Yeah, leftover after refactoring,